tests/test_networks.py

```diff
@@ -564,6 +565,34 @@ def test_build_url(kwargs, expected):
     assert AnyUrl(None, **kwargs) == expected
 
 
+@pytest.mark.parametrize(
+    'kwargs,expected',
+    [
+        (dict(scheme='http', host='example.net'), 'http://example.net'),
+        (dict(scheme='https', host='example.net'), 'https://example.net'),
+        (dict(scheme='http', user='foo', host='example.net'), 'http://foo@example.net'),
+        (dict(scheme='https', user='foo', host='example.net'), 'https://foo@example.net'),
```

Comment:
What about the cases of
```suggestion
        (dict(scheme='http', user='foo', host='example.net', port=80), 'http://foo@example.net'),
        (dict(scheme='https', user='foo', host='example.net', port=443), 'https://foo@example.net'),
```
?

Reply:
AnyHttpUrl does not hide port number, only HttpUrl does. Moved suggested cases to new test